addons/toolbars/src/components/ToolbarMenuButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Icons, IconButton } from '@storybook/components';
+
+import type { IconsProps } from '@storybook/components';
+
+interface ToolbarMenuButtonProps {
+  active: boolean;
+  title: string;
+  icon: IconsProps['icon'] | '';
+  description: string;
+  onClick?: () => void;
+}
+
+export const ToolbarMenuButton = ({
```

Comment:
How about?
```ts
export const ToolbarMenuButton: FC<ToolbarMenuButtonProps> = (
```

Reply:
My thought was after the talk to add FC<> when children or other FC based props are used and not use anything if nothing is required, but for now I will make it FC<> all over to make it even

Reply:
Fixed